lib/config/configuration.go

```diff
@@ -2054,6 +2054,10 @@ func Configure(clf *CommandLineFlags, cfg *service.Config, legacyAppFlags bool)
 		cfg.SetAuthServerAddress(cfg.Auth.ListenAddr)
 	}
 
+	if fileConf.Proxy.UiConfig.ScrollbackLength > 0 {
+		cfg.Proxy.UiConfig.ScrollbackLength = fileConf.Proxy.UiConfig.ScrollbackLength
+	}
+
```

Comment:
I'd really like some feedback on this one. Ideally I'd like to check if _any_ UiConfig is present and just set the entire thing that way, but haven't found a way to make it `nil` if nothing is set. Perhaps

Reply:
Since you have `yaml:ui,omitempty` you should be able to make the UI config a pointer in the fileconfig. The YAML unmarshaler will skip over it and leave it `nil` if not set.

Reply:
Awesome, this is exactly what I was looking for, thank you